packages/itwin/reports-config-widget/src/widget/components/ActionPanel.scss

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import '~@itwin/itwinui-css/scss/variables';
+
+.action-panel {
```

Comment:
use the `rcw` prefix for all your generic classnames, not going to manually comment this on every scss file